docs/model-archiver.md

```diff
@@ -159,4 +159,4 @@ For eagermode:
 torch-model-archiver --model-name densenet_161 --version 1.0 --model-file model.py --serialized-file model.pt --handler image_classifier
 ```
 
-This will package all the model artifacts files and output `densenet_161.mar` in the current working directory. This `.mar` file is all you need to run TorchServe, serving inference requests for a simple image recognition API. Go back to the [Serve a Model tutorial](../README.md#serve-a-model) and try to run this model archive that you just created!
+This will package all the model artifacts files and output `densenet_161.mar` in the current working directory. This `.mar` file is all you need to run TorchServe, serving inference requests for a simple image recognition API. Go back to the [Serve a Model tutorial](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model) and try to run this model archive that you just created!
```

Comment:
@jlin27 Can we keep all the links as relative?

Reply:
Hi Geeta - These were intentionally kept as absolute links because they point to the github repo, rather than documentation.